data/projects/create_jenkins_plugin.md

```diff
@@ -6,20 +6,35 @@ If you find this fun you'll be the one knowing this component best and will
 get the chance to maintain it on your own if you want - always with the
```

Comment:
i'd remove this paragraph. Instead add some details on how such plugin could be built (I have no idea but maybe using the json output for example). Anything else that could be helpful or interesting for the student also should be added.